bin/prettier.js

```diff
@@ -264,6 +266,11 @@ if (stdin) {
         if (output) {
           console.log(output);
         }
+      } else if (argv["list-different"]) {
+        if (input !== output) {
+          console.log(filename);
```

Comment:
@vjeux You're too fast! I was also working on the same issue 👍. Why not calling it simply `diff` instead of `list-different`? Regarding the printing, in gofmt's source (line 146) https://golang.org/src/cmd/gofmt/gofmt.go you can see that they use `fmt.Printf`.